bitcoin/src/consensus/validation.rs

```diff
@@ -146,12 +146,12 @@ impl Script {
     ///  * `flags` - Verification flags, see [`bitcoinconsensus::VERIFY_ALL_PRE_TAPROOT`] and similar.
     ///
     /// [`bitcoinconsensus::VERIFY_ALL_PRE_TAPROOT`]: https://docs.rs/bitcoinconsensus/0.106.0+26.0/bitcoinconsensus/constant.VERIFY_ALL_PRE_TAPROOT.html
-    pub fn verify_with_flags<F: Into<u32>>(
+    pub fn verify_with_flags(
         &self,
         index: usize,
         amount: crate::Amount,
         spending_tx: &[u8],
-        flags: F,
+        flags: impl Into<u32>,
```

Comment:
This is strictly a breaking change. I don't even know why there's this `Into<u32>` and why not have just `u32` or even better a newtype representing flags.

But I don't mind this change.

Reply:
Oh it is? I thought I read that this was syntax sugar for the exact same code it changes?

Reply:
Oh I see, one cannot use turbo fish now after this change so you are correct its a breaking change. I'll update the git log.

Reply:
oo, that's subtle. I also missed that this was breaking.